osutil/disks/disks.go

```diff
@@ -145,9 +145,16 @@ func parseUdevProperties(r io.Reader) (map[string]string, error) {
 	return m, scanner.Err()
 }
 
-// DiskFromName finds a matching Disk using the specified name, such as vda, or
-// mmcblk0, etc.
+// DiskFromDeviceName finds a matching Disk using the specified name, such as
+// vda, or mmcblk0, etc.
 func DiskFromDeviceName(deviceName string) (Disk, error) {
+	return diskFromDeviceName(deviceName)
+}
+
+// diskFromDeviceName is exposed for mocking from other tests via
+// MockDeviceNameDisksToPartitionMapping, but has the same problems as
+// diskFromMountPoint above, so we use a var here
```

Comment:
since it's actually below, maybe it's best to `s/above//`

Reply:
this is even simpler since it doesn't have the same problems